Successfully created express routes with the ability to deliver data to frontend

leonathompson opened the pull request.

leonathompson committed:
created backend boilerplate for testing; accidentally did this in the express branch

leonathompson committed:
Co-authored-by: mar467 <mar467@users.noreply.github.com>

leonathompson committed:
created routes for following endpoints: /admin, /prometheus, and /rp

leonathompson committed:
successfully created express routes with the ability to deliver data to the frontend as it should be

c-arriagada committed:
Delete server/package-lock.json

c-arriagada reviewed (changes requested):
Please test prometheus route.

leonathompson committed:
Verified prom router

leonathompson committed:
'Verified prom routes work'
Merge branch 'leona/express-routes' of https://github.com/oslabs-beta/KafHerder into leona/express-routes

leonathompson committed:
installed package-lock.json

c-arriagada reviewed (approved):
great job!
